wsmaster/codenvy-hosted-system-ram/src/main/java/com/codenvy/service/systemRam/SystemRamLimitStatus.java

```diff
@@ -0,0 +1,35 @@
+/*
+ *  [2012] - [2016] Codenvy, S.A.
+ *  All Rights Reserved.
+ *
+ * NOTICE:  All information contained herein is, and remains
+ * the property of Codenvy S.A. and its suppliers,
+ * if any.  The intellectual and technical concepts contained
+ * herein are proprietary to Codenvy S.A.
+ * and its suppliers and may be covered by U.S. and Foreign Patents,
+ * patents in process, and are protected by trade secret or copyright law.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained
+ * from Codenvy S.A..
+ */
+package com.codenvy.service.systemRam;
+
+/**
+ * Defines system RAM limit status.
+ *
+ * @author Igor Vinokur
+ */
+public enum SystemRamLimitStatus {
```

Comment:
Enum is possible but pure solution here, i suggest you to use separate object which provides dynamic information for that purpose, check the [comment](https://github.com/codenvy/codenvy/pull/501/files#r76417936) above